src/controllers/auth.controller.js

```diff
@@ -0,0 +1,47 @@
+const User = require("../models/User");
+const jwt = require("../service/jwtUtils");
+const errors = require("../constants/error");
+
+exports.login = async function (req, res, next) {
+  let member;
+
+  try {
+    const { email, username } = req.body;
+
+    member = await User.findOne({ email });
+
+    if (!member) {
+      member = await User.create({ email, username });
+    }
+
+    const accessToken = jwt.sign(member);
+    const refreshToken = jwt.refresh();
+
+    await User.findByIdAndUpdate(member._id, { refreshToken });
+
+    res
+      .status(201)
+      .cookie("AccessToken", accessToken, {
+        maxAge: 1000 * 60 * 60,
```

Comment:
쿠키의 maxAge를 1시간으로 설정해주셨군요.

해당 값을 상수화해서 코드 가독성을 높여보는 건 어떨까 제안드려봅니다!

```suggestion
        maxAge: ONE_HOUR_IN_MS,
```

Reply:
magic number의 활용 너무 좋네요! 바로 적용하겠습니다!